app/javascript/controllers/map_controller.js

```diff
@@ -37,7 +36,7 @@ export default class extends Controller {
     this.map.addControl(new mapboxgl.FullscreenControl(), 'top-left');
     this.map.addControl(new MapboxGeocoder({ accessToken: mapboxgl.accessToken,
       mapboxgl: mapboxgl }))
+    this.map.scrollZoom.disable();
+    this.map.dragPan.disable();
```

Comment:
removes ability to scroll the map and zoom it. This is so that when one is on mobile they don't accidentally scroll away from the country focussed, and are able to more easily scroll the page without scrolling the map by accident.